README.md

```diff
@@ -10,54 +9,70 @@ This is a community maintained Starter Project Battlesnake!
 Contribute to Open Source, and help keep this project up-to-date via pull request. Pull requests will be reviewed and merged by the [Battlesnake Official](https://github.com/BattlesnakeOfficial) team.
 
 Get involved in the Battlesnake community!
-* [Discord](https://play.battlesnake.com/discord)
-* [Twitch](https://www.twitch.tv/battlesnakeofficial)
+
+- [Discord](https://play.battlesnake.com/discord)
+- [Twitch](https://www.twitch.tv/battlesnakeofficial)
 
 ### Technologies
 
 This Battlesnake uses [Ruby 2.7](https://www.ruby-lang.org/), and [Heroku](https://heroku.com).
 
 ### Prerequisites
 
-* [GitHub Account](https://github.com/) and [Git Command Line](https://www.atlassian.com/git/tutorials/install-git)
-* [Heroku Account](https://signup.heroku.com/) and [Heroku Command Line](https://devcenter.heroku.com/categories/command-line)
-* [Battlesnake Account](https://play.battlesnake.com)
-
-
+- [GitHub Account](https://github.com/) and [Git Command Line](https://www.atlassian.com/git/tutorials/install-git)
+- [Heroku Account](https://signup.heroku.com/) and [Heroku Command Line](https://devcenter.heroku.com/categories/command-line)
+- [Battlesnake Account](https://play.battlesnake.com)
 
 ## Deploying Your First Battlesnake
 
 1. [Fork this repo](https://github.com/BattlesnakeOfficial/starter-snake-ruby/fork) into your GitHub Account.
 
 2. [Clone your forked repo](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) into your local environment.
-    ```shell
-    git clone git@github.com:[YOUR-GITHUB-USERNAME]/starter-snake-ruby.git
-    ```
+
+   ```shell
+   git clone git@github.com:[YOUR-GITHUB-USERNAME]/starter-snake-ruby.git
+   ```
 
 3. [Create a new Heroku app](https://devcenter.heroku.com/articles/creating-apps) to run your Battlesnake.
-    ```shell
-    heroku create [YOUR-APP-NAME]
-    ```
+   ```shell
+   heroku create [YOUR-APP-NAME]
+   ```
+   **Important:** The Starter Snake is built to use Ruby 2.7, but Heroku will default to Ruby 3.0 now as of May 2022.
```

Comment:
Can you remove this note since we're doing this by default with this change?

Reply:
Thank you. I wanted to confirm. Lines 37-40? Or line 40?